lib/puppet/reports/servicenow.rb

```diff
@@ -8,6 +8,57 @@
   def process
     settings_hash = settings
 
+    case settings['operation_mode']
+    when 'event_management'
+      process_event_management(settings_hash)
+    else
+      # default to incident management
+      process_incident_management(settings_hash)
+    end
+  rescue StandardError => e
+    Puppet.err "servicenow report processor error: #{e}\n#{e.backtrace}"
+  end
+
+  def process_event_management(settings_hash)
+    event_data = {
+      'source'   => 'Puppet',
+      'type'     => 'node_report',
+      # 5 => 'OK' severity
+      'severity' => '5',
```

Comment:
There's definitely some additional fields that could be set here (`additional_information`, for example), but I lean towards deciding those fields in a separate ticket.

Reply:
I was thinking additional information would be a good place to put the facts json, but yeah, merge this first and do that later.